cloudify_cli/config/parser_config.py

```diff
@@ -37,12 +37,24 @@ def blueprint_id_argument():
     }
 
 
+def snapshot_id_argument(hlp = 'The id of the snapshot'):
+    return {
+        'metavar': 'SNAPSHOT_ID',
+        'type': str,
+        'help': hlp,
+        'dest': 'snapshot_id',
+        'default': None,
```

Comment:
@idanmo, maybe we want to allow users to, by default, act upon the latest snapshot?